### DISCUSSION TITLE
Coffee heating tank, original thermistor and an SSR

### DISCUSSION THREAD

**Discussion by Dlloydev** [Dlloydev/sTune#9] [General]
@craig02445 

> I am trying to use sTune in my setup which is a Keurig coffee heating tank
with original thermistor and an SSR. The tank heats up very quickly (< 15
seconds to 100*º*C) but is very slow to cool. With an experimental setpoint
at 70 or 80 I get big overshoot with any setting (well, any I come up with)
and very long times to settle. I am even trying bringing it close to
temperature via other methods then invoking your code. I will gladly keep
playing and testing and get you any data that can be helpful to you (to
help others and me!), but can you guess at some good starting parameters
and settings for me to try?
Thanks for any suggestions.
-Craig
-------------------------------------------------
Sure,
I thought I'd try starting a new discussion (https://github.com/Dlloydev/sTune/discussions/9) so this could have a separate topic. If you could provide the part# or link to the SSR, a simple circuit diagram and your test code, that would be great. I'd be happy to take a look at it and help with the code and suggestions.
-David

**Comment by craig02445** [comment]
Great! I am using a [classic SSR](https://www.sparkfun.com/products/13015), simply wired in line with the 120v hot in series with the heater, safety thermostat and fuse that are in the machine. The thermistor is connected to an ADS1115 analog-to-digital converter. The thermistor has a nominal resistance of 59.1K, I _believe_ a bcoefficient of 3950, and I have a series resistor of 46.8K. 
I get reading of about 0x30F0 at room temperature and 0x5F30 close to boiling. And my previous statement of the speed to boiling was different conditions, a couple of minutes but again, cooling is much slower. I am working on some graphs now.
Using your TCLab sTune_QuickPID code.
